lib/qa/authorities/linked_data/find_term.rb

```diff
@@ -106,13 +111,19 @@ def consolidate_term_results(results) # rubocop:disable Metrics/MethodLength, Me
           end
           consolidated_results.each do |res|
             consolidated_hash = res[1]
-            consolidated_hash[:label] = sort_string_by_language consolidated_hash[:label]
-            consolidated_hash[:altlabel] = sort_string_by_language consolidated_hash[:altlabel]
-            consolidated_hash[:sort] = sort_string_by_language consolidated_hash[:sort]
+            consolidated_hash[:label] = sort_literals(consolidated_hash, :label)
+            consolidated_hash[:altlabel] = sort_literals(consolidated_hash, :altlabel)
+            consolidated_hash[:sort] = sort_literals(consolidated_hash, :sort)
           end
           consolidated_results
         end
 
+        def sort_literals(consolidated_hash, key)
+          return nil unless consolidated_hash.key? key
```

Comment:
This allows for the removal of `sort_string_by_language` from `rdf_helper` and uses the preferred language_sort_service as opposed to a method defined in an include module.